assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,84 @@
 ﻿<Root>
     <File Name="object_ob" Segment="6">
```

Comment:
Can you add a small comment explaining this object between the root and file nodes.